pkg/validation/test.go

```diff
@@ -879,6 +879,13 @@ func validateDNSConfig(fieldRoot string, dnsConfig []api.StepDNSConfig) (ret []e
 	return errs
 }
 
+func validateNodeArchitecture(fieldRoot string, nodeArchitecture api.NodeArchitecture) error {
+	if nodeArchitecture != api.NodeArchitectureAMD64 && nodeArchitecture != api.NodeArchitectureARM64 {
+		return fmt.Errorf("%s.nodeArchitecture expected one of %v or %v", fieldRoot, api.NodeArchitectureAMD64, api.NodeArchitectureARM64)
```

Comment:
But we are the ones that give value to the empty nodeSelectors. Can you elaborate more on this validation?

Reply:
This validation is to avoid someone accidentally using different or wrong architecture